docs/getting-started/server/events.md

```diff
@@ -205,4 +181,47 @@ exchange/queues.
    pwsh setup_secrets.ps1 -clear
    ```
 
-3. Start or re-start all services, including `EventsProcessor`.
+3. Start or re-start all services, including `EventsProcessor` (which is where the Azure Service Bus
+   listeners run)
+
+## Azure Queue
+
+The cloud instance of Bitwarden has migrated to using the Azure Service Bus implementation of
```

Comment:
⛏️ Strike this language about what we've done or rationale -- this should just document the Queue offering.